renderer/components/Form/PasswordInput.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import Input from './Input'
+import SvgEye from '../Icon/Eye'
+import SvgEyeOff from '../Icon/EyeOff'
+
+/**
+ * @name PasswordInput
+ * @example
+ * <PasswordInput />
+ */
+class PasswordInput extends React.Component {
+  state = {
+    isPasswordVisible: false,
+  }
+
+  toggleIsPasswordVisible = () => {
+    this.setState(prevState => ({
+      isPasswordVisible: !prevState.isPasswordVisible,
+    }))
+  }
+
+  render() {
+    const { isPasswordVisible } = this.state
+
+    return (
+      <Input
+        autoComplete="current-password"
+        css="position: relative;"
+        field="password"
+        isRequired
+        minLength={8}
```

Comment:
I thought about this and figured that since we use minLength={8} in all places but the seed phrase input I will set it a 8 as a starting point and nullify it as needed.

I'm OK with removing the minLength though.

Reply:
Done